packages/base/Checkbox/src/CheckboxGroup/CheckboxGroup.tsx

```diff
@@ -1,44 +1,35 @@
 /* eslint-disable react/no-array-index-key */
+import type { HTMLAttributes } from 'react'
 import React from 'react'
-import type { FormGroupProps } from '@material-ui/core'
-import { FormGroup } from '@material-ui/core'
-import type { Theme } from '@material-ui/core/styles'
-import { makeStyles, useTheme } from '@material-ui/core/styles'
-import cx from 'classnames'
 import type { GridSizeProps, GridProps } from '@toptal/picasso-grid'
 import { GridCompound as Grid } from '@toptal/picasso-grid'
-
-import styles from './styles'
+import { twMerge } from 'tailwind-merge'
 
 type GridSpacing = GridProps['spacing']
 
-export interface Props extends FormGroupProps, GridSizeProps {
+export interface Props extends HTMLAttributes<HTMLDivElement>, GridSizeProps {
   /** Align checkboxes horizontally  */
   horizontal?: boolean
   /** Defines amount of space between checkbox components (in px) */
   spacing?: GridSpacing
 }
 
-const useStyles = makeStyles<Theme>(styles, {
-  name: 'PicassoCheckboxGroup',
-})
-
 const CheckboxGroup = (props: Props) => {
   const { horizontal, spacing, xs, sm, md, lg, xl, className, ...rest } = props
 
-  const classes = useStyles()
-  const { spacing: themeSpacing } = useTheme()
-
   const direction = horizontal ? 'row' : 'column'
-  const gridSpacing = spacing ?? horizontal ? themeSpacing(2) : 0
+  const gridSpacing = spacing ?? horizontal ? 16 : 0
 
   const children = React.Children.toArray(rest.children)
 
   return (
-    <FormGroup
+    <div
       {...rest}
-      classes={classes}
-      className={cx(classes.root, className)}
+      className={twMerge(
+        'flex flex-col flex-wrap -mr-2 -mb-2',
+        horizontal ? 'flex-row' : undefined,
```

Comment:
NIT: We can simplify this line further
```suggestion
        horizontal && 'flex-row',
```